test/test.cpp

```diff
@@ -33,6 +33,7 @@
 using namespace tiny_dnn::activation;
 
 #include "test_network.h"
+// FIXME(Randl) switching previous and next lines breaks tests
```

Comment:
@pansk Probably connected to https://github.com/tiny-dnn/tiny-dnn/issues/514 . I'm trying to get more info now.